core/src/validator.rs

```diff
@@ -355,6 +357,7 @@ impl ValidatorConfig {
             enforce_ulimit_nofile: false,
             rpc_config: JsonRpcConfig::default_for_test(),
             block_production_method: BlockProductionMethod::default(),
+            enable_block_production_forwarding: true, // enable forwarding by default for tests
```

Comment:
Why? Isn't this the minority case?

Reply:
many local cluster tests rely on forwarding behavior